FirstHomework/ConsoleApp1/LibraryCount.cs

```diff
@@ -0,0 +1,27 @@
+﻿class LibraryCount
+{ 
+    public void MesAdd(int a, int result) // message if reader takes a book
```

Comment:
If you want to print specific messages - create separate class for it, you are breaking S principle here